.github/pipeline-descriptor.yml

```diff
@@ -20,60 +20,27 @@ docker_credentials:
   password: ${{ secrets.PAKETO_BUILDPACKS_DOCKERHUB_PASSWORD }}
 
 dependencies:
-- name:            JDK 11
-  id:              jdk
-  version_pattern: "11\\.[\\d]+\\.[\\d]+"
-  purl_pattern:    "11\\.[\\d]+\\.[\\d]+"
-  uses:            docker://ghcr.io/paketo-buildpacks/actions/graalvm-dependency:main
-  with:
-    glob:    graalvm-ce-java11-linux-amd64-.+.tar.gz
-    token:   ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
-    version: 11
-- name:            Native Image 11
-  id:              native-image-svm
-  version_pattern: "11\\.[\\d]+\\.[\\d]+"
-  cpe_pattern:     "[\\d]+\\.[\\d]+\\.[\\d]+\\.?[\\d]?"
-  purl_pattern:    "[\\d]+\\.[\\d]+\\.[\\d]+\\.?[\\d]?"
-  uses:            docker://ghcr.io/paketo-buildpacks/actions/graalvm-dependency:main
-  with:
-    glob:    native-image-installable-svm-java11-linux-amd64-.+.jar
-    token:   ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
-    version: 11
 - name:            JDK 17
   id:              jdk
   version_pattern: "17\\.[\\d]+\\.[\\d]+"
   purl_pattern:    "17\\.[\\d]+\\.[\\d]+"
-  uses:            docker://ghcr.io/paketo-buildpacks/actions/graalvm-dependency:main
+  uses:            docker://ghcr.io/paketo-buildpacks/actions/github-release-dependency:main
```

Comment:
Is there a reason you switched to github-release action for 17 and not for 20?

Reply:
Copy & paste error. :) Good catch. I'll fix that.